src/tools/rbtree.c

```diff
@@ -569,12 +569,16 @@ static rbnode *succ_node(rbnode *node) {
 }
 
 uint32_t rb_get(rbtree_t *tree, uintptr_t addr) {
+    if(tree->leftmost && addr < tree->leftmost->start) return 0;
+    if(tree->rightmost && addr > tree->rightmost->end) return 0;
     rbnode *node = find_addr(tree, addr);
     if (node) return node->data;
     else return 0;
```

Comment:
The 'else' is not necessary.

Reply:
It may be unnecessary, but it is a logical signal: the structure of the code is "if a node is found, do this, otherwise do that" (where "do" is actually "return"). It allows one to easily see at the first glance that the function will not always return `0`, and is similar to other functions that have a similar pattern. (Also, notice that it was not added by him, since there is no `+` on the leftmost column and it is not on a green background.)

Notice that without the `else`, the structure is:
```c
if (node) do_something;
return 0;
```
which is misleading at first glance.